src/NuGet.Core/NuGet.Packaging/Core/NuspecCoreReaderBase.cs

```diff
@@ -141,7 +141,7 @@ public virtual bool GetDevelopmentDependency()
         /// Returns a nuspec metadata value or string.Empty.
         /// </summary>
         public virtual string GetMetadataValue(string name)
-        {
+        {   
```

Comment:
Revert this, you are not making a change here.

Reply:
reverted